src/Networking/NexusMods.Networking.NexusWebApi/Auth/JWTToken.cs

```diff
@@ -63,7 +63,7 @@ public static Optional<EntityId> Create(IDb db, ITransaction tx, JwtTokenReply r
 
         tx.Add(entityId, AccessToken, reply.AccessToken);
         tx.Add(entityId, RefreshToken, reply.RefreshToken);
-        tx.Add(entityId, ExpiresAt, DateTimeOffset.FromUnixTimeSeconds(reply.CreatedAt).DateTime + TimeSpan.FromSeconds(reply.ExpiresIn));
+        tx.Add(entityId, ExpiresAt, DateTimeOffset.FromUnixTimeSeconds(reply.CreatedAt).UtcDateTime + TimeSpan.FromSeconds(reply.ExpiresIn));
```

Comment:
What is the reason for converting to a DateTime? Shouldn't we just keep using the DateTimeOffset directly?
```suggestion
        tx.Add(entityId, ExpiresAt, DateTimeOffset.FromUnixTimeSeconds(reply.CreatedAt)+ TimeSpan.FromSeconds(reply.ExpiresIn));
```